community/modules/scripts/wait-for-startup/main.tf

```diff
@@ -13,22 +13,40 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+locals {
+  instance_names = concat(var.instance_names, [var.instance_name])
```

Comment:
In case `var.instance_name` is not provided, how does `wait-for-startup-status.sh` handles empty(?) `INSTANCE_NAME`. 
I think better solution would be to deprecate `var.instance_name` in favor of `var.instance_names`

Reply:
Good point.  I've changed it to have a separate resource for each variable.  This should cover all error cases.